### PR TITLE
Known issues with AWS C2S

### PR DESCRIPTION
**Pull request by codyhoag**
Based on feedback in https://github.com/openshift/openshift-docs/issues/26801#issuecomment-781368519

Preview: https://deploy-preview-29614--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-known-issues

### CODE REVIEW

**Comment by staebler** [comment]
Looks great to me!

**Comment by codyhoag** [comment]
@yunjiang29 can you verify from a QE perspective? Thanks!

**Comment by yunjiang29** [comment]
@codyhoag LGTM. Thanks!

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -1205,6 +1203,10 @@ This script removes unauthenticated subjects from the following cluster role bin
 
 * {op-system} real time (RT) kernels are currently only supported on compute nodes, not control plane nodes. Compact clusters are not supported with RT kernels in {product-title} 4.7. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1887007[*BZ#1887007*])
 
+* It is currently not possible to use the technology preview AWS Secure Token Service (STS) in a cluster installed into the AWS C2S Secret Region due to current {product-title} limitations. This will be fixed in a future release of {product-title}. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1927157[*BZ#1927157*])
```
Comment (ahardin-rh) [inline release_notes/ocp-4-7-release-notes.adoc:1206]: use the AWS Secure Token Service (STS), which is a Technology Preview feature, in a cluster

**Review by jeana-redhat — APPROVED** [review]
approving exception

**Pull request merged**